### ISSUE TITLE
Chat bubble module -- more font options

### ISSUE THREAD

**Issue by mohcow** [Legacy-of-Sylvanaar/prat-3-0#104]
I'd like to see more font options added to the chat bubble module. Like being able to select the font freely, the possibility to make the font have an outline/thickoutline etc. This would make it easier to read when the backdrop is transparent.

**Comment by Rezy942** [comment]
Yes please. Even option to reset it to default blizz settings and let you configure from there.
